### PR TITLE
Handle email address in service request form & script

### PR DESCRIPTION
**Pull request by mddilley**
Closes 
https://github.com/cityofaustin/atd-knack-dts-portal/issues/4

### CODE REVIEW

**Comment by mddilley** [comment]
This PR adds handling of data collected in the Knack DTS Portal form that we don't want displayed in GitHub. I added a `quote_text_hidden` format type and some handling of this type to the `merge` method block. 

@johnclary - how have you approached testing this bot? I didn't want to create a mess of GitHub issues. Also, what about the Knack side of this issue? I haven't implemented form val on the Knack side but can look into it if needed.

**Comment by johnclary** [comment]
thanks @mddilley! great question. i haven't had to test since we deployed the prod bot. i will create a test environment fo the knack app and add an env param to the script.

File: intake.py [hunk 1]
```diff
@@ -329,7 +335,8 @@ def main():
             # update knack record as "Sent" using form API, which will
             # trigger an email notificaiton
 
-            token = get_token(KNACK_USERNAME, KNACK_PASSWORD, KNACK_APP["app_id"])
+            token = get_token(KNACK_USERNAME, KNACK_PASSWORD,
+                              KNACK_APP["app_id"])
```
Comment (johnclary) [inline intake.py]: @mddilley this formatting looks good, but i'm curious if you're using a linter. If you haven't already, would you run this through [Black](https://github.com/psf/black)? lmk if you need help with setup.

**Comment by johnclary** [comment]
@mddilley code looks beautiful. will hold off on approving until we can test.

File: intake.py (hunk 1 above)
Comment (mddilley) [reply intake.py]: I had not heard about Black - thanks for sharing! I just ran the code through and pushed along with another fix.

**Comment by mddilley** [comment]
The bot email handling has now been tested. Screenshot below. The email field validation for austintexas.gov has also been added to the `test` and `prod` environments in Knack.

### GH description in edit mode (email hidden when not editing)
Screen Shot 2019-11-26 at 3 51 48 PM

**Comment by johnclary** [comment]
awesome @mddilley! let's hold off on merge and do that when we deploy next week.

**Comment by johnclary** [comment]
some light reading for you on our deployment framework. if this gets you wishing we were using airflow instead, great.
https://github.com/cityofaustin/atd-data-deploy

**Comment by johnclary** [comment]
@mddilley we're all set. this branch is running happily. merging and refreshing on `atd-data01`. thanks!

**Pull request merged**